src/components/POIForm.component.js

```diff
@@ -1,9 +1,215 @@
 import React, { Component } from 'react'
+import {
+  FormGroup,
+  FormControl,
+  ControlLabel,
+  Button,
+  Checkbox,
+  Form,
+  Col,
+  PageHeader
+} from 'react-bootstrap'
+import DatePicker from 'react-datepicker'
+import moment from 'moment'
+import 'react-datepicker/dist/react-datepicker.css'
 
 class POIForm extends Component {
+  constructor(props) {
+    super(props)
+    this.onDateSelected = this.onDateSelected.bind(this)
+    this.state = {
+      startDate: moment(),
+      name: '',
+      description: '',
+      selectedStories: [],
+      links: ''
+    }
+    this.handleChange = this.handleChange.bind(this)
+    this.onChangeName = this.onChangeName.bind(this)
+    this.onChangeDescription = this.onChangeDescription.bind(this)
+    this.onChangeLinks = this.onChangeLinks.bind(this)
+    /*this.onClickPlus = this.onClickPlus.bind(this)*/
+    this.onSubmit = this.onSubmit.bind(this)
+  }
+
+  onDateSelected() {
+    // call this when date input changes
+  }
+
+  handleChange(date) {
+    this.setState({
+      startDate: date
```

Comment:
This is really doing what you want `onDateSelected` to do right? Why not move this into that function, since `onDateSelected` is more descriptive than `handleChange`? And don't forget to update the prop appropriately in the `DatePicker` component